pasqal_cloud/__init__.py

```diff
@@ -20,7 +20,12 @@
 from pasqal_cloud.batch import Batch, RESULT_POLLING_INTERVAL
 from pasqal_cloud.client import Client
 from pasqal_cloud.device import BaseConfig, EmulatorType
-from pasqal_cloud.endpoints import AUTH0_CONFIG, Auth0Conf, Endpoints, PASQAL_ENDPOINTS
+from pasqal_cloud.endpoints import (  # noqa: F401
```

Comment:
as the endpoints and auth0 config is not used directly here but imported by the users, using the noqa flag prevent flake8 errors